README.md

```diff
@@ -1,26 +1,293 @@
 # Resussun
 
-## How to start the Resussun application
+Resussun is an implementation of the [Reconciliation Service API](#https://reconciliation-api.github.io/specs/0.1/), 
+providing reconciliation endpoints for datasets from [Timbuctoo](#https://github.com/HuygensING/timbuctoo) instances.
+
+1. [Running the service](#running-the-service)
+    1. [Use a local build](#use-a-local-build)
+    1. [Use docker-compose.yml](#use-docker-composeyml)
+    1. [Use docker-compose-local.yml](#use-docker-compose-localyml)
+1. [How the service is built up](#how-the-service-is-built-up)
+1. [Mapping and searching](#mapping-and-searching)
+    1. [How the data is mapped and searched](#how-the-data-is-mapped-and-searched)
+    1. [An example](#an-example)
+1. [API](#api)
+    1. [Web interface](#web-interface)
+    1. [Admin interface](#admin-interface)
+
+## Running the service
 
 ### Use a local build
 1. Run `mvn clean install` to build your application
 1. Start application with `./target/appassembler/bin/resussun server config.yml`
 1. To check that your application is running enter url `http://localhost:8080`
 
 ### Use docker-compose.yml
-1. run the docker-compose file: `docker-compose -f docker/docker-compose.yml up -d`  
+1. Run the docker-compose file: `docker-compose -f docker/docker-compose.yml up -d`  
 1. To check that your application is running enter url `http://localhost:8080`
 
 ### Use docker-compose-local.yml
 This docker-compose uses the local build to run with Elasticsearch and Redis.
 1. Run `mvn clean install` to build your application
-1. run the docker-compose file: `docker-compose -f docker/docker-compose-local.yml up -d`  
+1. Run the docker-compose file: `docker-compose -f docker/docker-compose-local.yml up -d`  
 1. To check that your application is running enter url `http://localhost:8080`
 
-### Create an index
-1. Create an index: `curl -X POST --data "timbuctooUrl={timbuctooUrl}&dataSetId={dataSetId}" http://{resussun_host}:{admin_port}/tasks/createIndex`
-For localhost use `http://localhost:8081/tasks/createIndex`
+## How the service is built up
+
+Resussun is a service written in Java using the [Dropwizard](#https://www.dropwizard.io) framework. 
+It makes use of [ElasticSearch](#https://www.elastic.co/elasticsearch) as the search engine 
+and [Redis](#https://redis.io) for the storage of metadata and to provide caching.
+
+![](./docs/components.png)
+
+Through the admin interface an administrator can create reconciliation services for any number of configured datasets 
+from various Timbuctoo instances. Resussun will use the [GraphQL](#https://graphql.org) interface 
```

Comment:
We might want to add a link to Timbuctoo.

Reply:
Link added.

Reply:
Just now I see that the external links do not work. The `#` should be removed.